app/javascript/src/Policies/middleware/PolicyChain.jsx

```diff
@@ -43,10 +43,10 @@ const loadChain = ({registry, storedChain, dispatch}: {registry: Array<RegistryP
   })
   if (errors > 0) {
     dispatch(loadChainError({})) // TODO: Define what to do with errors (unlikely now, just undefined path returned by Array.find)
-  } else {
```

Comment:
Can the errors something else then policy error?

Reply:
Errors are only and exactly when:
`policy.name === storedPolicy.name && policy.version === storedPolicy.version`